### DISCUSSION TITLE
Improving README.md in the Quickstart section

### DISCUSSION THREAD

**Discussion by layer7gmbh** [netbox-community/netbox-docker#406] [General]
Hi, 

at very first, thank you for your work! Using netbox with docker makes setup in general much easier/faster.

Since i run into this issue, i wanted to suggest to change the Quickstart information in the readme.md of the project page.

Currently its:

```
git clone -b release https://github.com/netbox-community/netbox-docker.git
cd netbox-docker
tee docker-compose.override.yml <<EOF
version: '3.4'
services:
  nginx:
    ports:
      - 8000:8080
EOF
docker-compose pull
docker-compose up
```

But without defining:

ALLOWED_HOSTS 

like:     ALLOWED_HOSTS=localhost ::1 127.0.0.1

in the netbox.env file inside the env folder of the downloaded package

**before**

running 

```docker-compose up```

the brought up container will answer browser connection attempts with

```The connection was reset```

So defining ALLOWED_HOSTS is actually mandatory.

Without doing it, this:

"The whole application will be available after a few minutes. Open the URL http://0.0.0.0:8000/ in a web-browser."

will always fail ( at least it did always fail with me ). By the way should that 0.0.0.0 not be better replaced by "localhost" ?

So in my humble opinion, this could be mentioned in the Quickstart section to make things more streight forward.

Greetings & stay healthy!
Oliver

**Comment by cimnine** [comment]
If `ALLOWED_HOSTS` is undefined, we consider it as `'*'`, which allows any hostname, as per [Netbox documentation](https://netbox.readthedocs.io/en/stable/installation/3-netbox/#allowed_hosts).

You can see this in our [`configuration.py`](https://github.com/netbox-community/netbox-docker/blob/aa4d630a0fa423c757635cdb95f97e3389bb6ff4/configuration/configuration.py#L37).
